hub/demo/src/lib/entries.tsx

```diff
@@ -119,6 +119,10 @@ export function idForEntry(entry: z.infer<typeof entryModel>) {
   return `${entry.namespace}/${entry.name}/${entry.version}`;
 }
 
+export function idMatchesEntry(id: string, entry: z.infer<typeof entryModel>) {
+  return id.startsWith(`${entry.namespace}/${entry.name}`);
```

Comment:
Maybe we need the / in here to make it more precise. Though it would be an interesting feature to have a "" named assistant agent to allow permissions for any agent in a namespace.

Reply:
Good point! I'll push a commit here to add `/` at the end.